system/src/system_network_internal.h

```diff
@@ -387,7 +386,7 @@ class ManagedNetworkInterface : public NetworkInterface
             WLAN_CONNECTED = 0;
             WLAN_DHCP = 0;
 
-            cloud_disconnect();
+            cloud_disconnect_graceful();
```

Comment:
I don't remember exactly what we decided on when discussing futures and graceful disconnection, but if I remember correctly the idea was that `WiFi.disconnect()` and `Cellular.disconnect()` calls should close the connection sanely and disconnect from WiFi or cellular network, compared to `WiFi.off()` and `Cellular.off()` which would terminate the connection and turn off wireless module 'immediately'.

What would be the cons of closing cloud connection normally before disconnecting from wireless network?